src/nyc_trees/apps/event/models.py

```diff
@@ -231,6 +231,16 @@ def my_events_now(cls, user):
 
         attended = [r.event for r in registrations if r.did_attend]
         non_attended = [r.event for r in registrations if not r.did_attend]
+
+        # Group admins don't need to RSVP to be able to survey events.
+        # These filters are copy & pasted from the query directly above.
+        admin_events = Event.objects \
+            .filter(group__admin=user, begins_at__gte=beginning_of_day_utc) \
+            .filter(Q(ends_at__lte=end_of_day_utc) |
+                    Q(begins_at__lte=now_utc + STARTING_SOON_WINDOW,
+                      ends_at__gt=now_utc))
```

Comment:
No need to do this now if you don't want to, but FYI a nice way to avoid this duplication is to do something like this:

```python
events_soon = Event.objects \
    .filter(begins_at__gte=beginning_of_day_utc) \
    .filter(Q(ends_at__lte=end_of_day_utc) |
            Q(begins_at__lte=now_utc + STARTING_SOON_WINDOW,
              ends_at__gt=now_utc))

registrations = EventRegistration.objects \
    .filter(user=user) \
    .filter(event_id__in=events_soon.values_list('id', flat=True))

attended = [r.event for r in registrations if r.did_attend]
non_attended = [r.event for r in registrations if not r.did_attend]

attended = attended + list(events_soon.filter(group__admin=user))
```